### PR TITLE
Rewrite conclusion

### PR DESCRIPTION
**Pull request by danich1**
Edited the conclusion section to reflect new conclusions within this paper rewrite.

### CODE REVIEW

**Review by dhimmel — APPROVED** [review]

File: content/06.conclusion_future_direction.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
 ## Conclusion and Future Direction
 
 Filling out knowledge bases via manual curation can be an arduous and erroneous task [@doi:10.1093/bioinformatics/btm229].
-As the rate of publications increases manual curation becomes an infeasible approach.
+As the rate of publications increases manual curation alone becomes an impractical approach.
```
Comment (cgreene) [inline content/06.conclusion_future_direction.md]:
```suggestion
As the rate of publications increases, relying on manual curation alone becomes impractical.
```

File: content/06.conclusion_future_direction.md [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
 Data programming, a paradigm that uses label functions as a means to speed up the annotation process, can be used as a solution for this problem.
-A problem with this paradigm is that creating a useful label function takes a significant amount of time. 
+An obstabcle this paradigm faces is that creating a useful label function takes a considerable amount of time.
```
Comment (cgreene) [inline content/06.conclusion_future_direction.md]:
```suggestion
An obstacle for this paradigm is creating useful label functions, which takes a considerable amount of time. 
```

File: content/06.conclusion_future_direction.md [hunk 3]
```diff
@@ -1,18 +1,17 @@
 ## Conclusion and Future Direction
 
 Filling out knowledge bases via manual curation can be an arduous and erroneous task [@doi:10.1093/bioinformatics/btm229].
-As the rate of publications increases manual curation becomes an infeasible approach.
+As the rate of publications increases manual curation alone becomes an impractical approach.
 Data programming, a paradigm that uses label functions as a means to speed up the annotation process, can be used as a solution for this problem.
-A problem with this paradigm is that creating a useful label function takes a significant amount of time. 
+An obstabcle this paradigm faces is that creating a useful label function takes a considerable amount of time. 
 We tested the feasibility of reusing label functions as a way to speed up the  label function creation process.
```
Comment (cgreene) [inline content/06.conclusion_future_direction.md]:
```suggestion
We tested the feasibility of reusing label functions as a way to reduce the total number of label functions required for strong prediction performance.
```

File: content/06.conclusion_future_direction.md [hunk 4]
```diff
@@ -1,18 +1,17 @@
...
-We conclude that label function re-use across edge types can increase performance when there are certain constraints on the number of functions re-used.
-More sophisticated methods of reuse may be able to capture many of the advantages and avoid many of the drawbacks.
-Adding more relevant label functions can increase overall performance.
-The discriminative model, under this paradigm, has a tendency to overfit to predictions of the generative model.
-We recommend implementing regularization techniques such as drop out and weight decay to combat this issue.
+We conclude that label function can be re-used with closely related edge types (Compound binds Gene and Gene interacts Gene), but is practically futile for distantly related ones (Compound treats Disease and Disease associates Gene).
```
Comment (cgreene) [inline content/06.conclusion_future_direction.md]:
```suggestion
We conclude that label functions may be re-used with closely related edge types, but that re-use does not improve performance for most pairings.
```

File: content/06.conclusion_future_direction.md [hunk 5]
```diff
@@ -1,18 +1,17 @@
...
+More sophisticated methods may be able to connect distantly related edge types and avoid many of the drawbacks we encountered in this work.
```
Comment (cgreene) [inline content/06.conclusion_future_direction.md]:
```suggestion
```

File: content/06.conclusion_future_direction.md [hunk 6]
```diff
@@ -1,18 +1,17 @@
...
+The discriminative model's performance improves as more edge-specific label functions are incorporated into the generative model; however, we did notice that performance greatly depends on the generativce model.
```
Comment (cgreene) [inline content/06.conclusion_future_direction.md]:
```suggestion
The discriminative model's performance improves as more edge-specific label functions are incorporated into the generative model; however, we did notice that performance greatly depends on the generative model.
```

File: content/06.conclusion_future_direction.md [hunk 7]
```diff
@@ -1,18 +1,17 @@
...
+We recommend that future endeavors focus on efficiently tuning the generative model to ensure proper training of the discriminative model.
```
Comment (cgreene) [inline content/06.conclusion_future_direction.md]:
```suggestion
```

File: content/06.conclusion_future_direction.md [hunk 8]
```diff
@@ -1,18 +1,17 @@
...
 
 This work sets up the foundation for creating a common framework that mines text to create edges.
-Within this framework we would continuously ingest new knowledge as novel findings are published, while providing a single confidence score for an edge by consolidating sentence scores.
-Different from existing hetnets like Hetionet where text-derived edges generally cannot be exactly attributed to excerpts from literature [@doi:10.7554/eLife.26726; @doi:10.15363/thinklab.d67], our approach would annotate each edge with its source sentences.
+Within this framework we would continuously ingest new knowledge as novel findings are published, while providing a single confidence score for an edge via sentence score consolidation.
+Different from existing hetnets like Hetionet where text-derived edges generally cannot be exactly attributed to excerpts from literature [@doi:10.7554/eLife.26726; @doi:10.15363/thinklab.d67], our approach would annotate each edge based on its source sentences.
```
Comment (cgreene) [inline content/06.conclusion_future_direction.md]:
```suggestion
As opposed to many existing knowledge graphs, for example Hetionet where text-derived edges generally cannot be exactly attributed to excerpts from literature [@doi:10.7554/eLife.26726; @doi:10.15363/thinklab.d67], our approach has the potential to annotate each edge based on its source sentences.
```

File: content/06.conclusion_future_direction.md [hunk 9]
```diff
@@ -1,18 +1,17 @@
...
 In addition, edges generated with this approach would be unencumbered from upstream licensing or copyright restrictions, enabling openly licensed hetnets at a scale not previously possible [@doi:10.15363/thinklab.d107; @doi:10.1038/536016a; @doi:10.1371/journal.pone.0213090].
 Accordingly, we plan to use this framework to create a robust multi-edge extractor via multitask learning [@doi:10.1145/3209889.3209898] to construct continuously updating literature-derived hetnets.
```
Comment (cgreene) [inline content/06.conclusion_future_direction.md]:
```suggestion
New multitask learning [@doi:10.1145/3209889.3209898] strategies may make it even more practical to reuse label functions to construct continuously updating literature-derived knowledge graphs.
```

**Review by cgreene — APPROVED** [review]
I have a few suggestions - looking good!

**Pull request merged**